src/js/views/chargeMoneyView.ts

```diff
@@ -0,0 +1,57 @@
+import { $ } from '../utils/common';
+import { chargeMoneyTemplate, sectionTemplate } from '../templates/chareMoneyTemplate';
+import { validateInputMoney } from '../validates/validates';
+import { CoinsType } from '../types/types';
+import { SELECTOR, CUSTOM_EVENT } from '../constants/constants';
+
+export default class ChargeMoneyView {
+  $content: HTMLDivElement;
+
+  constructor() {
```

Comment:
```suggestion
  constructor(private readonly vendingMachine: VendingMachine) {
```
Vending Machine을 직접 생성자로 받아서 처리하면 어떨까요?

Reply:
뷰에서 바로 vendingMachine에 데이터를 가져다 쓰고, 넣어주고 하는 것으로 이해했습니다. 적용해봤는데 컨트롤러가 필요없게 되더라고요! 정확히 이해를 못한것 같아서 저가 이해한게 맞을까요?

Reply:
넵 이해하신게 맞아요 🙂 
매번 controller에서 넘겨주는거보단 생성자에서 vendingMachine을 넣어주는게 관리하기 편할거같아 추천드렸어요!

Reply:
넵! 그러면 이번에 다시 적용해보겠습니다. 👍🏻